charts/ai-service/Chart.yaml

```diff
@@ -5,7 +5,7 @@ maintainers:
   - name: unique-ag
     url: https://unique.ch/
 name: ai-service
-version: 1.0.8
+version: 1.0.9
 description: |
   The 'ai-service' chart is a "convenience" chart from Unique AG that can generically be used to deploy simple AI workloads to Kubernetes.
 
```

Comment:
We can also add a changes block:
https://github.com/Unique-AG/helm-charts/blob/f133a7a375e263b46c4e76b5f44f461de5a3a2e6/charts/web-app/Chart.yaml#L15

And maybe, please change the reference icon:
https://github.com/Unique-AG/helm-charts/blob/f133a7a375e263b46c4e76b5f44f461de5a3a2e6/charts/web-app/Chart.yaml#L3

And add a source:
https://github.com/Unique-AG/helm-charts/blob/f133a7a375e263b46c4e76b5f44f461de5a3a2e6/charts/web-app/Chart.yaml#L13

Reply:
Added source and changes. What kind of icon would you like to display here?

Reply:
Same as in the other charts, the caumasee, I linked it above

Reply:
Ah, it was already there. Adapted the changes structure and moved sources further above under name.